_posts/2016-10-15-recurse-center-week-3.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: post
+title: "Week[2] @ The Recurse Center 🌻"
+comments: true
+categories: tech recurse
+---
+
+1/4 of the way through! Time is flying.
+This post is also quite late. I suppose it's a good sign that I've been too busy to write.
+
+# Overall
+
+The week started off as a low, and steadily improved to wonderful levels. 😊
+
+### Emotional 😱
+
+#### **I felt emotionally drained.**
+
+At the start of the week I felt pretty emotionally drained.
+
+My past experiences with making connections/getting involved with communities involved a lot of putting myself out there and stepping out of my comfort zone.
+It's a bit of a numbers game. The more you do these things, the higher your chances of finding those people you click with. At the same time, it can go overboard.
+
+It can become forced.
+It can turn into a lack of boundaries.
+It can have the opposite effect.
+
+When you over invest, you can go bankrupt. Investments need space/time to grow.
+
+#### **I questioned the value of my offers.**
+
+A sticky trap I fall into is self-doubt when my expectations of others don't line up with how things play out in real life.
+I actually have had only positive interactions with people in person. Yet when I was extending offers for events/activities, their was less enthusiasm then expected.
+
+I started wondering if my offers had value. Or whether the quality of these interactions were as good as I thought.
+
+#### **I wondered if RC was wrong for me.**
+
+Another thought was that maybe RC just wasn't compatible with me.
+I actually got rejected the first time I applied. Not even after an interview...right after the application.
+
+Maybe my social goals weren't aligned with this space.
+Maybe RC is more focused on the technical growth.
+Maybe I didn't belong.
+
+#### **I focused on things outside the space.**
+
+I realized there was a lot for me to explore outside of RC. As a result, no evenings were spent 100% at RC last week.
+
+I went to shows, tech events and meet-ups. This relieved a lot of the pressure I was putting on RC. Instead of expecting RC to fulfill all my needs, I was being proactive in seeking out alternative spaces. It helped me disassociate a bit and regain some sense of autonomy.
```

Comment:
Interesting question!

A few things:
- I find friendships need to be built in a variety of contexts. If your relationships with people only takes place in a single context, then it's hard to say that those are friendships vs. situational relationships. Whereas when you spend time with people in a variety of contexts, then you can be more certain that everyone enjoys each others company vs. being forced into each others company by the context.
- By taking myself out of RC and seeing who else would come along, it helped me distinguish who actually wants to be friends vs. who I just have passing conversations with because we are in the same space.
- By taking myself out of the space I wasn't forcing my brain in programming mode 100% of the time.
- RC was a default for me. I told myself I should be there rather then sit at home. By researching other options I showed myself there's more options than RC or sitting at home.
- In RC there's a lot of focus on our programmer identity. "What are you working on?", "Where have you been applying for jobs?", "Who wants to pair?". When I left I got a much better sense of who people actually were. We could actually talk about RC and share our experiences (something that can be awkward to do when you're in the space).

...I might be able to go on.

It would be interesting to add this stuff...or maybe I could write about it next week... need to code :P.

Thanks for the question! Hoped that made sense :).